lib/improver/metadata/attributes.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2019 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""
+Module defining attributes for IMPROVER blended data and the nowcast
+"""
+
+import re
+import warnings
+
+from improver.utilities.cube_metadata import amend_metadata
+
+
+STANDARD_GRID_TITLE_STRING = "UK 2 km Standard Grid"
+SPOT_TITLE_STRING = "UK Spot Values"
+
+DATASET_ATTRIBUTES = {
+    "nowcast": {
+        "source": "IMPROVER",
+        "title": "MONOW Extrapolation Nowcast",
+        "institution": "Met Office"
+    },
+    "multi-model blend": {
+        "source": "IMPROVER",
+        "title": "IMPROVER Post-Processed Multi-Model Blend",
+        "institution": "Met Office"
+    }
+}
```

Comment:
I would prefer the default set of attributes to be separated from the code that implements the changes. Whilst I can reconcile myself to not using a more configurable option (such as a JSON file), I think we should try to separate the setting of "constants" from the code making the changes; these can easily be imported. This would reflect what we have done with unit and datatypes.

Reply:
There are a couple of reasons I did it this way, but I can see your point.  I'm going to do a little reading around how to structure python modules and think about the best solution here.

Reply:
I have now restructured this to have a "constants" subpackage in the metadata package.  This is where I will want to put "units.py" in the next PR, and can be used to store any other top level metadata constants we want to set at a later date.  The functions to apply the attribute changes are in the package top level, in the renamed "set_attributes" module.